src/main/scala/service/db/CardTableService.scala

```diff
@@ -0,0 +1,27 @@
+package service.db
+
+import cats.effect.{Async, Resource}
+import cats.syntax.all._
+import domain.Card
+import domain.ID._
+import slick.jdbc.PostgresProfile.api._
+import slickeffect.Transactor
+
+trait CardTableService[F[_]] {
+  def getAllCardsByGameId(gameId: GameId): F[Seq[Card]]
+  def getCardById(id: CardId): F[Option[Card]]
+  def insert(cards: Seq[Card]): F[Unit]
+  def update(card: Card): F[Unit]
+}
+object CardTableService {
```

Comment:
```suggestion
object CardTableDbService {
```
Not quite self explanatory, that service covers db layer operation